Feature/doc restructure installation (#11)
microROS/micro-ROS-doc: Installation/README.md

﻿# Development tools installation

> This section will provide all steps of how install all required development tools for building a MicroROS workspace.

Reviewer: ```suggestion
This section will provide all steps of how to install all required development tools for building a MicroROS workspace.
```

The development tools for Micro-ROS in Linux platforms is the same as the one for [ROS2](https://github.com/ros2/ros2/wiki), so the same set-up is needed.

> All required tool that is going to be installed are been extracted from the [ROS2 istallation guide](https://index.ros.org/doc/ros2/Linux-Install-Debians/).

Reviewer: ```suggestion
All required tool that is going to be installed have been extracted from the [ROS2 installation guide](https://index.ros.org/doc/ros2/Linux-Install-Debians/).
```

1. [Windows platform](Windows_DevTools.md)
1. [Linux platform](Linux_DevTools.md)
1. [Nuttx platform (Using Linux)](NuttxLinux_DevTools.md)

# ROS2/Micro-ROS Workspace build and install 

> This section will provide all steps of how build and install a Micro-ROS workspace. 

Reviewer: ```suggestion
This section will provide all steps of how to build and install a Micro-ROS workspace. 
```
